static/app/views/monitors/components/overviewTimeline/monitorEnvironmentLabel.tsx

```diff
@@ -1,26 +1,54 @@
 import styled from '@emotion/styled';
 
 import {Tooltip} from 'sentry/components/tooltip';
+import {IconFix, IconMute} from 'sentry/icons';
+import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import type {ColorOrAlias} from 'sentry/utils/theme';
```

Comment:
probbaly won't need this import once you drop the cast

Reply:
ahh rip still do for the styled component :(